### PR TITLE
I2-PayFineUi

### PR DESCRIPTION
**Pull request by haynescrossley**
@elliotmitchum PayFineUI  class ready for review before merging into master.

Branched directly from latest master

### CODE REVIEW

**Comment by elliotmitchum** [comment]
@amrmosit This is ready for your review

File: src/library/payfine/PayFineUI.java [hunk 1]
```diff
@@ -5,115 +5,115 @@
...
+                    control.cardSwiped(patronId);
                 }
                 catch (NumberFormatException e) {
-                    DiSpLaYoUtPuT("Invalid patronID");
+                    displayOutput("Invalid patronID");
                 }
                 break;
                 
             case PAYING:
                 double AmouNT = 0;
```
Comment (amrmosit) [inline src/library/payfine/PayFineUI.java]: PR 45 D1 CSG#2 All variable names are to start with a lowercase letter and to be in camelBack

File: src/library/payfine/PayFineUI.java [hunk 2]
```diff
@@ -5,115 +5,115 @@
...
-                String Amt_Str = GeTiNpUt("Enter amount (<Enter> cancels) : ");
+                String Amt_Str = getInput("Enter amount (<Enter> cancels) : ");
```
Comment (amrmosit) [inline src/library/payfine/PayFineUI.java]: PR 45 D2 CSG#2 All variable names are to start with a lowercase letter and to be in camelBack

**Review by amrmosit — CHANGES_REQUESTED** [review]
@haynescrossley 
Please there are two defects to be fixed.

File: src/library/payfine/PayFineUI.java (hunk 1 above)
Comment (haynescrossley) [reply src/library/payfine/PayFineUI.java]: [a976b79](https://github.com/elliotmitchum/ITC205/pull/45/commits/a976b792b0d29fabe9934709b5a268adf7f27cd5)

File: src/library/payfine/PayFineUI.java (hunk 2 above)
Comment (haynescrossley) [reply src/library/payfine/PayFineUI.java]: [0931b27](https://github.com/elliotmitchum/ITC205/pull/45/commits/0931b27482e30530c04d653380ec0cfe35966618)

**Comment by haynescrossley** [comment]
@amrmosit resolved those defects. Over to you to double check

**Review by amrmosit — APPROVED** [review]
@haynescrossley 
all good. Thanks
@elliotmitchum branch review is approved.

**Pull request merged**